wunderio/drupal-project
# Added files generated by Drupal scaffolding to .gitignore.

**Changes proposed**

- Add files generated by Drupal scaffolding to .gitignore

**Motivation**

Scaffolding files can be altered in various ways, [as described here](https://www.drupal.org/docs/develop/using-composer/using-drupals-composer-scaffold#toc_4), so there should be no need to commit them to version control. If that would be needed, git .gitignore can be adjusted per project to allow that.

**To test**

1. Do clean clone of `master` branch
1. Run `composer install`
1. Run `git status` and notice you have a bunch of untracked files, not very nice
1. Repeat using this branch, notice the files aren't untracked since they are gitignored
